config/constants.js

```diff
@@ -0,0 +1,6 @@
+const breakpointKeys = ['a', 'b', 'c', 'd', 'e', 'f']
+
+module.exports = {
+   breakpointKeys,
+   responsiveStrFragmentRegexPattern: `(.*?)\\[([${breakpointKeys.join('')}-]+)\\]`
```

Comment:
Suggestion: maybe set `const responsiveStrFragmentRegexPattern = `(.*?)\\[([${breakpointKeys.join('')}-]+)\\]`` on line 2 for consistency?